Caesar-Cipher/caesar_cipher.py

```diff
@@ -0,0 +1,30 @@
+alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u',

```

Comment:
Instead of this manual work, use the string module to retrieve them. (string.ascii_lowercase) or (string.ascii_uppercase)

Reply:
Done.